### DISCUSSION TITLE
Getting requests.exceptions.HTTPError: 500 trying to create sharepoint list item

### DISCUSSION THREAD

**Discussion by fullstackpoole** [O365/python-o365#1024] [Q&A]
I've created a Sharepoint list and can get items, update items, and delete items on it without issue. When I call sp_list.create_list_item() and pass a dictionary with the exact column structure, I get the HTTP 500 error: Error Message: General exception while processing. Following the link the error gives me, I get: {"error":{"code":"InvalidAuthenticationToken","message":"Access token is empty."}}. Anyone have any ideas what's going on?

**Comment by alejcas** [comment]
Can't help you here sorry
